cnxeasybake/scripts/main.py

```diff
@@ -22,6 +23,12 @@ def easybake(css_in, html_in=sys.stdin, html_out=sys.stdout, last_step=None):
     # serialize out HTML
     print (etree.tostring(html_doc, method="html"), file=html_out)
 
+    # generate CSS coverage_file file
+    if coverage_file:
+        print('SF:{}'.format(css_in.name), file=coverage_file)
```

Comment:
from all the lcov files I've looked at (`genhtml`, JS coverage tools) they all write the absolute path and then strip out the common parts of the path. It would also help if/when the `*.lcov` files are stored somewhere other than where the script is run from (relative paths become a problem then).